meinberlin/apps/cms/models/pages.py

```diff
@@ -82,7 +82,7 @@ class HomePage(Page):
 class DocsPage(Page):
     body = fields.StreamField(
         [
-            ("documents_list", cms_blocks.DocsBlock()),
+            ("documents_list", cms_blocks.AccordionBlock()),
```

Comment:
should we change this to `accordion_list` so the element in wagtail is also called that?